src/internal/minikubetestenv/deploy.go

```diff
@@ -78,6 +131,14 @@ func UpgradeRelease(t *testing.T, ctx context.Context, kubeClient *kube.Clientse
 	return testutil.AuthenticatedPachClient(t, testutil.NewPachClient(t), user)
 }
 
+func InstallReleaseEnterprise(t *testing.T, ctx context.Context, kubeClient *kube.Clientset, user string) *client.APIClient {
+	createSecretEnterpriseKeySecret(t, ctx, kubeClient, ns)
+	opts := union(localDeploymentWithMinioOptions(ns, localImage), withEnterprise(t, ns))
+	require.NoError(t, helm.InstallE(t, opts, helmChartLocalPath, helmRelease))
+	waitForPachd(t, ctx, kubeClient, ns, localImage)
+	return testutil.AuthenticatedPachClientPostActivate(t, testutil.NewPachClient(t), user)
+}
+
 func DeleteRelease(t *testing.T, ctx context.Context, kubeClient *kube.Clientset) {
```

Comment:
Let's not expose this, and instead register cleanup in InstallReleaseEnterprise, with `t.Cleanup`, so it is cleaned up at the end of the test.